superflore/generators/ebuild/ebuild.py

```diff
@@ -157,13 +157,11 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
         ret += " license\n\n"
 
         # EAPI=<eapi>
-        ret += "EAPI=" + self.eapi + "\n\n"
-        """
-        @todo: don't hard code this
-        """
+        ret += "EAPI=" + self.eapi + "\n"
+        ret += "PYTHON_COMPAT=( python{2_7,3_5} )\n\n"
```

Comment:
Maybe, but that should be handled on a case-by-base basis, as I don't have a list available. Do you see any reason that this should not be merged, aside from this? This is not meant to be a permanent fix, but rather another step along the journey.